pt/endpoints/powerdata/model.py

```diff
@@ -9,133 +11,113 @@
+    #         "history_id": history_id,
+    #         "address": address,
+    #     }
+    #     super(EV, self).__init__(mother)
+    #     self.cluster = data_struct["cluster"]
+    #     self.power_display = data_struct["power"]
 
 
 class PV(PowerData):
     __tablename__ = "pv"
-    uuid = db.Column(db.String(40), db.ForeignKey("powerdata.uuid"), primary_key=True)
+    uuid = db.Column(UUID(), db.ForeignKey("powerdata.uuid"), primary_key=True)
     cluster = db.Column(db.Integer)
     pac = db.Column(db.Float)
     __mapper_args__ = {"polymorphic_identity": "PV"}
 
-    def __init__(self, data_struct, history_id, address):
-        mother = {
-            "uuid": data_struct["id"],
-            "field": data_struct["field"],
-            "updated_at": data_struct["updated_at"],
-            "history_id": history_id,
-            "address": address,
-        }
-        super(PV, self).__init__(mother)
-        self.cluster = data_struct["cluster"]
-        # pylint: disable=C0103
-        self.pac = data_struct["PAC"]
-        # pylint: enable=C0103
+    # def __init__(self, data_struct, history_id, address):
+    #     mother = {
+    #         "uuid": data_struct["id"],
+    #         "field": data_struct["field"],
+    #         "updated_at": data_struct["updated_at"],
+    #         "history_id": history_id,
+    #         "address": address,
+    #     }
+    #     super(PV, self).__init__(mother)
+    #     self.cluster = data_struct["cluster"]
+    #     # pylint: disable=C0103
+    #     self.pac = data_struct["PAC"]
+    #     # pylint: enable=C0103
 
 
 class WT(PowerData):
     __tablename__ = "wt"
-    uuid = db.Column(db.String(40), db.ForeignKey("powerdata.uuid"), primary_key=True)
+    uuid = db.Column(UUID(), db.ForeignKey("powerdata.uuid"), primary_key=True)
     cluster = db.Column(db.Integer)
     windgridpower = db.Column(db.Float)
     __mapper_args__ = {"polymorphic_identity": "WT"}
 
-    def __init__(self, data_struct, history_id, address):
-        mother = {
-            "uuid": data_struct["id"],
-            "field": data_struct["field"],
-            "updated_at": data_struct["updated_at"],
-            "history_id": history_id,
-            "address": address,
-        }
-        super(WT, self).__init__(mother)
-        self.cluster = data_struct["cluster"]
-        # pylint: disable=C0103
-        self.windgridpower = data_struct["WindGridPower"]
-        # pylint: enable=C0103
+    # def __init__(self, data_struct, history_id, address):
+    #     mother = {
+    #         "uuid": data_struct["id"],
+    #         "field": data_struct["field"],
+    #         "updated_at": data_struct["updated_at"],
+    #         "history_id": history_id,
+    #         "address": address,
+    #     }
+    #     super(WT, self).__init__(mother)
+    #     self.cluster = data_struct["cluster"]
+    #     # pylint: disable=C0103
+    #     self.windgridpower = data_struct["WindGridPower"]
+    #     # pylint: enable=C0103
```

Comment:
------, We can remove this section instead of commenting it.

Reply:
This is not the latest version, all comments have been remove :)